src/components/custom/_dbanner.tsx

```diff
@@ -1,13 +1,19 @@
 import React from 'react'
 import styled from 'styled-components'
-import PropTypes from 'prop-types'
+import type { ImageDataLike, IGatsbyImageData } from 'gatsby-plugin-image'
```

Comment:
`ImageDataLike` type is enough here since `IgatsbyImageData` is a part of `ImageDataLike`.